HidPkg/UefiHidDxeV2/src/keyboard.rs

```diff
@@ -393,14 +393,34 @@ impl KeyboardHidHandler {
         Ok(())
     }
 
+    /// Called to send HID reports to the device.
+    fn send_output_reports(
+        &mut self,
+        hid_io: &dyn HidIo,
+        output_reports: Vec<(Option<ReportId>, Vec<u8>)>,
+    ) -> Result<(), efi::Status> {
+        for (id, output_report) in output_reports {
+            let result = hid_io.set_output_report(id.map(|x| u32::from(x) as u8), &output_report);
+            if let Err(result) = result {
+                debugln!(
+                    DEBUG_ERROR,
+                    "KeyboardHidHandler:set_output_report: unexpected error sending output report: {:?}",
+                    result
+                );
+                return Err(result);
+            }
+        }
+        Ok(())
+    }
+
     /// Resets the keyboard driver state. Clears any pending key state. `extended verification` will also reset toggle
     /// state.
-    pub fn reset(&mut self, hid_io: &dyn HidIo, extended_verification: bool) -> Result<(), efi::Status> {
+    pub fn reset(&mut self, _hid_io: &dyn HidIo, extended_verification: bool) -> Result<(), efi::Status> {

```

Comment:
consider just removing `_hid_io` as I don't think this is an external API, so that wouldn't be a breaking change.

Reply:
Removed.